src/main/src/main/java/org/geoserver/ows/LocalWorkspaceCatalogFilter.java

```diff
@@ -79,15 +80,35 @@ public boolean hideStyle(StyleInfo style) {
         return hideWorkspace(style.getWorkspace());
     }
 
+    /**
+     * Should local workspaces include layer groups from the global workspace
+     * @return
+     */
+    public static boolean workspaceLayerGroupInherit() {
+        String value = GeoServerExtensions.getProperty("GEOSERVER_GLOBAL_LAYER_GROUP_INHERIT");
```

Comment:
The property cannot really be changed at runtime can it? So why not put it in a static field and avoid the repeated lookups.